configs/Gigabyte/GA-MA770-UD3.conf

```diff
@@ -0,0 +1,92 @@
+# Configuration file for Gigabyte GA-MA770-UD3 motherboard
+# GA-MA770-UD3.conf
+
+# By Clipo
+# 2/11/2019
+
+# Based on config file for GA-MA770-DS3 contributed by Charles A. from India.
+
+chip "it8720-*"
+
+    # Labels are copied from BIOS setup screens when value shown
+
+    # Voltages
+
+    label   in0 "Vcore"
+		# No scaling so no compute
+    set     in0_min 1.4 * 0.95  # 95% of nominal min
+    set     in0_max 1.4 * 1.05  # 105% of nominal max
+
+    label   in1 "DDR2 1.8V"
+		# No scaling so no compute
+    set     in1_min 1.8 * 0.95 # 95% of nominal
+    set     in1_max 1.8 * 1.05 # 105% of nominal
+
+    label   in2 "+3.3V"
+		# No scaling so no compute
+    set     in2_min 3.3 * 0.95 # 95% of nominal
+    set     in2_max 3.3 * 1.05 # 105% of nominal
+
+    label   in3 "+5V"
+		# Assume standard Winbond scaling resistors
+    compute in3 @ * ((6.8/10)+1), @ / ((6.8/10)+1)
+    set     in3_min 5 * 0.95 # 95% of nominal
+    set     in3_max 5 * 1.05 # 105% of nominal
+
+    ignore  in4  # Value never changed from 3.10
+
+    ignore  in5  # Value fluctuated by to much to be a valid voltage
+
+    label   in6 "+12V"
+    		# Assume standard Winbond scaling resistors
+    compute in6 @ * ((28/10)+1), @ / ((28/10)+1)
+    set     in6_min 12 * 0.95 # 95% of nominal
+    set     in6_max 12 * 1.05 # 105% of nominal
+    
+    label  in7 "VCCH5V" # dmesg reported In7 set from VCCH5V
+    compute in7 @ * ((6.8/10)+1), @ / ((6.8/10)+1)
+    set     in7_min 5 * 0.95 # 95% of nominal
+    set     in7_max 5 * 1.05 # 105% of nominal 
+
+    label   in8 "Vbat 3V CR2032"
+		# Connected directly so no compute
+    #in8_min and _max not supported by chip
+
+    ignore  cpu0_vid # Not dynamic under lm-sensors 3.0.0
+
+    # Temperatures
+
+    label   temp1  "Current System Temperature"
+		# BIOS: 32
+    compute temp1 @ -5, @ +5 # Correction for what seems to be an in accurate thermal sensor 
+    set     temp1_min 0
+    set     temp1_max 45 # Tropical ambient max plus a bit
+
+    label   temp2  "Current CPU Casing Temperature"     
+		# BIOS: 30
+    set     temp2_min 0
+		# Max is lowest available BIOS "CPU Warning Temperature"
+    set     temp2_max 55 
+
+    ignore  temp3
+		# Values seen: 77-81, rising from cold start.
+		# Fanning the Northbridge did not change value enough to indicate cooling.
+
+    ignore intrusion0
```

Comment:
If this is the intrusion header of the board, the `ignore` should be commented out in the upstream config. Some users may want to use it, so let's not hide it from them.